recipes/ckanweb-deploy-exts.rb

```diff
@@ -389,6 +406,18 @@
 		command "rm -f /etc/cron.*/ckan-harvest*"
 	end
 end
+if not data_qld_harvester_present then
```

Comment:
This is unnecessary, since the configuration file is wiped and replaced from the cookbook before running this recipe.

Reply:
does it hurt leaving it in just in case there is extra items to go in here?